packages/jest-config/src/normalize.js

```diff
@@ -64,6 +64,11 @@ const setupPreset = (
     // $FlowFixMe
     preset = (require(presetModule): InitialOptions);
   } catch (error) {
+    if (error instanceof SyntaxError) {
```

Comment:
I'd rather flip it.

```js
if (error.code === 'MODULE_NOT_FOUND') {
  throw createConfigError(`  Preset ${chalk.bold(presetPath)} not found.`);
}

throw createConfigError(
  `  Unable to load preset ${chalk.bold(presetPath)}:\n  ${error.message}`,
);
```

Reply:
@SimenB I am getting a different error for that (AssertionError for the actual require), the error code will only be shown in test (MODULE_NOT_FOUND from jest-resolve).

Reply:
Oh, interesting. Thanks for checking!